api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionEmailAddress.java

```diff
@@ -0,0 +1,49 @@
+package org.pmiops.workbench.db.model;
+
+import javax.persistence.*;
```

Comment:
spotless might ding you here, but there's an IJ setting somewhere to let you turn this off for everything except a whitelist, e.g. Mockito.

Reply:
Does that help, though?  We can't merge if spotless doesn't like it.

Reply:
Never mind.  Was able to fix.